packages/epo-widget-lib/src/widgets/CameraFilter/CameraFilter.tsx

```diff
@@ -0,0 +1,123 @@
+import { FunctionComponent, useState } from "react";
+import { SelectListbox, tokens } from "@rubin-epo/epo-react-lib";
+import { useTranslation } from "react-i18next";
+import useResizeObserver from "use-resize-observer";
+import { filters, Filter, rangeConfig } from "./data";
+import * as Styled from "./styles";
+import CondensedFilterRanges from "./CondensedFilterRanges";
+import SpectrumDisplay from "./SpectrumDisplay";
+
+const CameraFilter: FunctionComponent = () => {
+  const { t } = useTranslation();
+  const { ref, width = 1 } = useResizeObserver<HTMLDivElement>();
+  const [activeFilterBand, setActiveFilterBand] = useState<string | null>(null);
+  const { BREAK_TABLET } = tokens;
+  const isCondensed = width < parseInt(BREAK_TABLET);
+
+  const {
+    [isCondensed ? "condensed" : "default"]: { min, max, range: spectrumRange },
+  } = rangeConfig;
```

Comment:
responsiveness is mostly done by setting two range configs for narrow or wide containers and selecting the display values based on the current container size.